CONTRIBUTING.md

```diff
@@ -2,6 +2,11 @@
 
 First time contributing to Homebrew? Read our [Code of Conduct](https://github.com/Homebrew/.github/blob/HEAD/CODE_OF_CONDUCT.md#code-of-conduct).
 
+If you are creating a pull request, ensure your commits follow the
```

Comment:
This is mentioned in "To add a new formula" below and `brew bump-formula-pr` will handle it for you too. Thoughts?

Reply:
I still pull out my "please use homebrew/core style in your commit messages" saved reply at least once a day, so I think a lot of users still miss it.

I'm fine with making this less verbose (i.e. back to @cdayjr's original suggestion 😅) so easier to scan.